ibis-server/app/model/metadata/redshift.py

```diff
@@ -104,7 +104,7 @@ def get_constraints(self) -> list[Constraint]:
         # we reuse the connector.query method to execute the SQL
         # so we have to give a limit to avoid too many rows
         # the default limit for constraints metadata is 500, i think it's a sensible limit
-        response = self.connector.query(sql, limit=500).to_pylist()
+        response = self.connector.query(sql, limit=50000).to_pylist()
```

Comment:
```suggestion
        response = self.connector.query(sql).to_pylist()
```
An alternative is setting no limit for this query. This SQL has only 9 columns. I think it's affordable to accept more than 50k rows.

Reply:
Thanks @goldmedal , I'd assumed based on the comments above the query call that `limit` was a required field. I've removed the limit from both queries and cleaned up the comments suggesting a limit is required.